dask/array/tests/test_reductions.py

```diff
@@ -67,6 +67,8 @@ def test_reductions_1D(dtype):
     reduction_1d_test(da.nansum, a, np.nansum, x)
     with ignoring(AttributeError):
         reduction_1d_test(da.nanprod, a, np.nanprod, x)
+        reduction_1d_test(da.nancumprod, a, np.nancumprod, x)
```

Comment:
Partially, I did add the backport but am still failing on CI for python < 3.4.  However, I think I've set this up quite right just yet.  Please let me take another look...